bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:tapo:device">
+		<parameter name="ipAddress" type="text" required="true">
+			<context>network-address</context>
+			<label>IP Address</label>
+		</parameter>
+		<parameter name="pollingInterval" type="integer" min="0" max="9999" required="false">
+			<label>Refresh Interval</label>
+			<description>Refresh interval for refreshing the data in seconds. (0=disabled)</description>
+			<default>30</default>
+			<advanced>true</advanced>
+		</parameter>
+	</config-description>
+
+	<config-description uri="bridge-type:tapo:bridge">
+		<parameter name="username" type="text" required="true">
+			<context>email</context>
+			<label>Username</label>
+			<description>Tapo-Cloud Login User (e-Mail)</description>
+		</parameter>
+		<parameter name="password" type="text" required="true">
+			<context>password</context>
+			<label>Password</label>
+			<description>Tapo-Cloud Login Password</description>
+		</parameter>
+		<parameter name="cloudReconnect" type="integer" min="1" max="168" required="false">
+			<label>Cloud reconnect interval</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions